encoder/MysqlEncoder.php

```diff
@@ -0,0 +1,204 @@
+<?php
+/**
+ * @link https://github.com/tom--/dynamic-ar
+ * @copyright Copyright (c) 2015 Spinitron LLC
+ * @license http://opensource.org/licenses/ISC
+ */
+
+namespace spinitron\dynamicAr\encoder;
+
+use spinitron\dynamicAr\ValueExpression;
+
+class MysqlEncoder extends BaseEncoder
```

Comment:
`MariaEncoder`. MySQL is going to be different

I read the docs for MySQL JSON features and it should work